hail/src/main/scala/is/hail/methods/IBD.scala

```diff
@@ -3,7 +3,7 @@ package is.hail.methods
 import is.hail.HailContext
 import is.hail.annotations._
 import is.hail.expr.ir._
-import is.hail.expr.types.physical.PString
+import is.hail.expr.types.physical.{PFloat64, PInt64, PString, PStruct}
```

Comment:
Obviously all #6416

Reply:
Again, not sure why merge didn't modify the diff: https://github.com/hail-is/hail/pull/6416/files#diff-974b03f89566a9d87c87b58f59987942R6